pyhap/accessory_driver.py

```diff
@@ -96,6 +99,32 @@ class HAP_CONSTANTS:
     SERVICE_COMMUNICATION_FAILURE = -70402
 
 
+class AIOThread(threading.Thread):
+    """
+    TODO: temporary convenience class. Create an event loop and event object for
+    controlling the Accessory.run method.
+    """
+
+    def __init__(self, run_method):
+        """
+        """
+        self.loop = asyncio.new_event_loop()
+        self.stop_event = asyncio.Event()
+        self.task = self.loop.create_task(run_method(self.stop_event, self.loop))
+        super(AIOThread, self).__init__(target=self.loop.run_until_complete,
```

Comment:
There is no need for `super(AIOThread, self).__init__(`, this can be shorted to `super().__init__(` in Python 3 and higher.